extensions/amp-list/amp-list.md

```diff
@@ -220,6 +220,23 @@ In several cases, we may need the `<amp-list>` to resize on user interaction. Fo
 </amp-list>
 ```
 
+### Initialization from amp-state
+
+In some cases, it may be desirable to have your `<amp-list>` component initialize off of `<amp-state>` rather than from a json endpoint.
```

Comment:
Let's be more prescriptive here to help developers do the right thing. When is this a good idea?

Reply:
Added some more detail, let me know if you think it can be improved further!